README.md

```diff
@@ -22,57 +30,92 @@
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-# 📖 [your_project_name] <a name="about-project">Hello Microverse</a>
 
-**[your_project__name]** This is a basic HTML & CSS project to practice all the tools that a wiil need to use in the future.
+# 📖 [Hello Microverse project] <a name="about-project"></a>
+
+
+**[Hello Microverse project]** is a basic Html and Css project made to practice with all the tools we will need ahead in the program. 
 
 ## 🛠 Built With <a name="built-with"></a>
 
+HTML5, CSS, Git and Github
 ### Tech Stack <a name="tech-stack"></a>
 
```

Comment:
- [ ] I assume that your PR can be more descriptive than this... The "file modified" question has been answered by the "Files Changed" tab. 

<img width="391" alt="image" src="https://user-images.githubusercontent.com/41163571/234767991-099ed56f-72e1-4229-9b39-c54426e8e322.png">


Kindly state what you did in those files. 😉